doc/ref_cert/RC1/chapters/chapter02.md

```diff
@@ -173,8 +140,7 @@ The NFVI Conformance framework deals with the process of testing NFVI in below t
 
 The NFVI KPIs are compared with Golden KPIs, which serve as a reference for NFVI Conformance. If NFVI KPIs meet Golden KPIs, NFVI is certified and granted a Conformance badge. If NFVI KPIs do not meet Golden KPIs, no Conformance badge is provided.
 
-<a name="2.7.1"></a>
-### 2.7.1 Best Practices (General)
+### Best Practices (General)
 The NFVI Conformance framework will be guided by the following core principles:
```

Comment:
I think there are other places in these docs where the NFVI has to be replaced with "the Cloud Infrastructure"